substrate/frame/identity/src/lib.rs

```diff
@@ -387,6 +442,12 @@ pub mod pallet {
 		/// A dangling username (as in, a username corresponding to an account that has removed its
 		/// identity) has been removed.
 		DanglingUsernameRemoved { who: T::AccountId, username: Username<T> },
+		/// A username has been unbound.
+		UsernameUnbound { username: Username<T> },
+		/// A username has been removed.
+		UsernameRemoved { username: Username<T> },
+		/// A username has been killed.
+		UsernameKilled { username: Username<T> },
```

Comment:
`UsernameRemoved` and `UsernameKilled` sounds very synonymous. May be should call it as `ForceUsernameRemoved`?

Reply:
From pallet terminology in docs, "removed" and "killed" are different things. I'd prefer to stick to even naming convention of 2 word events and defer to call documentation. I could write more comprehensive docs for the event enum variants.